examples/scripts/chat.py

```diff
@@ -0,0 +1,322 @@
+import copy
+import json
+import os
+import pwd
+import re
+import time
+from threading import Thread
+
+import torch
+from rich.console import Console
+from rich.live import Live
+from rich.markdown import Markdown
+from transformers import AutoModelForCausalLM, AutoTokenizer, TextIteratorStreamer
+
+from trl.commands.cli_utils import ChatArguments, TrlParser, init_zero_verbose
+from trl.trainer.utils import get_kbit_device_map, get_quantization_config
+
+
+init_zero_verbose()
```

Comment:
I think you need to import that before transformers and call it before any transformers import - make sure to put a `# flake8: noqa` at the top of the file to make the CI happy: https://github.com/huggingface/trl/blob/6cfa5cfc817be0f8acf30f8158a4a0ea244dc988/examples/scripts/sft.py#L1